### PR TITLE
WIP blog about Galaxy at PASC

### PR DESCRIPTION
**Pull request by hrhotz**

### CODE REVIEW

File: content/news/2023-07-07-PASC-conference/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Sergey Yakubov (Oak Ridge National Laboratory, USA) was presenting [**Lessons Learned Deploying Galaxy at a Neutron Scattering Facility**](https://pasc23.pasc-conference.org/presentation/?id=msa158&sess=sess169).
+
+Rolf Verberg (Cornell High Energy Synchrotron Source, USA) was presenting [**Transitioning Existing Data Reduction Workflows at the Cornell High Energy Synchrotron Source to Galaxy**](https://pasc23.pasc-conference.org/presentation/?id=msa221&sess=sess169).
+
+Hans-Rudolf Hotz (Friedrich Miescher Institute, CH) was presenting [**The Galaxy Training Network: A Powerful Framework for Teaching Science**](https://pasc23.pasc-conference.org/presentation/?id=msa165&sess=sess169).
+
+Leandro Liborio (Science and Technology Facilities Council, UK) was presenting [**Galaxy and Scientific Communities: How to Incorporate New Ones, Uplift Engaged Ones and Maintain Mature Ones**](https://pasc23.pasc-conference.org/presentation/?id=msa156&sess=sess169), describing several examples from the [**EuroScienceGateway project**](https://galaxyproject.org/projects/esg/).
+
+
+ <img src="IMG_0701.jpg" alt="workshop presenters" width="500">
```
Comment (bgruening) [inline content/news/2023-07-07-PASC-conference/index.md:21]:
```suggestion
 <img src="./IMG_0701.jpg" alt="workshop presenters" width="500">
```

File: content/news/2023-07-07-PASC-conference/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ 
+(Sergey, on-line; Leandro; Greg; Rolf; Hans-Rudolf)  
+
+ <img src="IMG_0685.jpg" alt="conference dinner" width="500">
```
Comment (bgruening) [inline content/news/2023-07-07-PASC-conference/index.md:25]:
```suggestion
 <img src="./IMG_0685.jpg" alt="conference dinner" width="500">
```

File: content/news/2023-07-07-PASC-conference/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ 
+(catching up with Volodymyr Savchenko (École Polytechnique Fédérale de Lausanne, CH) another collaborator from the [**EuroScienceGateway project**](https://galaxyproject.org/projects/esg/), see his recent blog post on 
+[**Galaxy for Astronomy: Integration of the FITS file format**](https://galaxyproject.org/news/2023-06-20-esg-wp5-astronomy-fits/) )
+
+ <img src="IMG_0704.jpg" alt="greg interview" width="500">
```
Comment (bgruening) [inline content/news/2023-07-07-PASC-conference/index.md:30]:
```suggestion
 <img src="./IMG_0704.jpg" alt="Greg interview" width="500">
```

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@dannon do you know why it does not find the images?

> #### http://localhost:8080/news/2023-07-07-pasc-conference/
> - [ ] ./IMG_0701.jpg
> - [ ] ./IMG_0685.jpg
> - [ ] ./IMG_0704.jpg

**Comment by bgruening** [comment]
@hrhotz could you please turn this one image?

**Comment by volodymyrss** [comment]
Look great, thanks @hrhotz!

**Comment by hrhotz** [comment]
@bgruening 
eh?? I am pretty sure that image was correct originally - did this happen as part of the 'automagically compressed' step?

**Comment by hrhotz** [comment]
@bgruening I have exchanged the picture (new: "IMG_0701.2.jpg"). It has been correct, and now after "Calibre's image-actions" has worked on it as well, it is turned again.

excuse my ignorance: do you have any ideas on how to revert this?

**Comment by dannon** [comment]
Sorry I missed the ping here, will try to see what's going on locally

**Comment by dannon** [comment]
Should be good to go now.  Yes, the compression action was actually rotating the image.  I manually compressed that, fixed the rotation, then rebased.

**Comment by bgruening** [comment]
Thanks!

**Comment by hrhotz** [comment]
@dannon thank you very much for your help !

**Pull request merged**